docs/adr/0005-graduation of EventBridge.md

```diff
@@ -0,0 +1,57 @@
+# 5. Graduation of EventBridge in CAPA
+
+* Status: [proposed]
+* Date: TBD 
+* Authors: @Ankitasw
+* Deciders: @richardcase @sedefsavas
+
+## Context
+EventBridge is an eventbus provided by AWS to watch for the events generated by any SaaS application running on AWS, or any of the AWS services, that could be used by any other services.
+
+With respect to CAPA, EventBridge helps watch EC2 instance state change events using which _AWSMachine_ controller would trigger reconciliation and handle the lifecycle of the EC2 instances.
+
+There are other `CloudWatch` events that exists which can be watched upon and CAPA controllers can take necessary actions based on the events, for eg. ASG Scale-in events, Spot Instance Termination Notices, Scheduled Maintenance events etc.
+
+## Decision
```

Comment:
An ADR is used to record a lightweight decision and the consequences of this decision. I'm struggling to understand what is the decision here from the description.

For me the decision is "We will graduate the EventBridge support out of experimental so that it is GA available and enabled by default in CAPA because we want to be able to use the different event types in the future."

Reply:
Makes sense to me, Will add this in decision